### PR TITLE
feat:add progress functionality to the details page.

### PR DESCRIPTION
**Pull request by wangxinbiao**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it
add progress functionality to the details page.
add 'top_p,' 'temperature,' 'max_tokens,' and 'prompt_template' parameters during task processing.
The details page displays information about QA segmentation models.

#### Which issue(s) this PR fixes

Fixes #

### CODE REVIEW

File: data-processing/data_manipulation/file_handle/common_handle.py [hunk 1]
```diff
@@ -663,4 +749,104 @@ def _convert_support_type_to_map(supprt_type):
     for item in supprt_type:
         result[item['type']] = item
 
-    return result
\ No newline at end of file
+    return result
+
+def _update_document_staus_and_start_time(
```
Comment (nkwangleiGIT) [inline data-processing/data_manipulation/file_handle/common_handle.py:754]: _update_document_staus_and_start_time => _update_document_**status**_and_start_time

File: data-processing/db-scripts/init-database-schema.sql [hunk 2]
```diff
@@ -96,3 +96,39 @@
     COMMENT ON COLUMN public.data_process_task_question_answer.update_user IS '更新用户';
     COMMENT ON COLUMN public.data_process_task_question_answer.update_program IS '更新程序';
 
+    CREATE TABLE IF NOT EXISTS public.data_process_task_document
```
Comment (nkwangleiGIT) [inline data-processing/db-scripts/init-database-schema.sql:99]: 注意这个 sql 文件的注释：
    -- ****************************************************************************
    -- If any update to this file, MUST sync the content to
    -- configMap under deploy/charts/arcadia/pg-init-data-configmap.yaml
    -- ****************************************************************************

File: data-processing/data_manipulation/file_handle/common_handle.py (hunk 1 above)
Comment (wangxinbiao) [reply data-processing/data_manipulation/file_handle/common_handle.py:754]: 好的

File: data-processing/db-scripts/init-database-schema.sql (hunk 2 above)
Comment (wangxinbiao) [reply data-processing/db-scripts/init-database-schema.sql:99]: 已添加进去

**Comment by bjwswang** [comment]
@wangxinbiao 数据处理读取minio数据源信息时，处理`insecure`有问题。需要考虑`insecure`为None的情况

**Comment by bjwswang** [comment]
@wangxinbiao 👍

File: apiserver/graph/schema/dataprocessing.graphqls [hunk 3]
```diff
@@ -180,10 +180,22 @@ type DataProcessConfigChildren {
   enable: String
   zh_name: String
   description: String
+  llm_config: LLMConfig
   preview: [DataProcessConfigpreView]
   file_progress: [DataProcessConfigpreFileProgress]
 }
 
+type LLMConfig {
```
Comment (Abirdcfly) [inline apiserver/graph/schema/dataprocessing.graphqls:188]: should we also update `apiserver/pkg/dataprocessing/dataprocessing.go` file to handle this type?

Comment (wangxinbiao) [reply apiserver/graph/schema/dataprocessing.graphqls:188]: not necessary, this file is just for making HTTP requests.

**Pull request merged**